src/components/v2/V2Create/index.tsx

```diff
@@ -26,7 +25,7 @@ export default function V2Create() {
     <V2UserProvider>
       <V2WarningBanner />
       <div style={{ margin: '4rem', marginBottom: 0 }}>
-        <h1>
+        <h1 style={{ marginBottom: '2rem', color: 'rgba(0,0,0,0.933' }}>
```

Comment:
**Suggestion:** Do you think we could add this color to the themes like `colors.text.tabHeader` or something?

Reply:
i was trying to use same style as var(--text-primary), but i don't know how to use that in here. LOL 
Do you know how should i do it?

Reply:
Oh yep, get the colors from the `ThemeContext` like this: `const { colors } = useContext(ThemeContext).theme`. Then go `colors.text.primary`

Reply:
ahhh got it, awesome, thanks so much Johnny!